rmk-macro/src/keycode_alias.rs

```diff
@@ -4,343 +4,247 @@ use once_cell::sync::Lazy;
+    add_alias!("Dot" = ".");
+    add_alias!("Slash" = "/");
+    add_alias!("CapsLock" = "caps_lock", "caps");
+    add_alias!("F1");
+    add_alias!("F2");
+    add_alias!("F3");
+    add_alias!("F4");
+    add_alias!("F5");
+    add_alias!("F6");
+    add_alias!("F7");
+    add_alias!("F8");
+    add_alias!("F9");
+    add_alias!("F10");
+    add_alias!("F11");
+    add_alias!("F12");
+    add_alias!("PrintScreen" = "print_screen", "pscr");
+    add_alias!("ScrollLock" = "scroll_lock", "scrl");
+    add_alias!("Pause" = "paus", "brk", "brmu");
+    add_alias!("Insert" = "ins");
+    add_alias!("Home");
+    add_alias!("End");
+    add_alias!("PageUp" = "page_up", "pgup");
+    add_alias!("PageDown" = "page_down", "pgdn");
+    add_alias!("Delete" = "del");
+    add_alias!("Right" = "rght", "->", "→");
+    add_alias!("Left" = "<-", "←");
+    add_alias!("Down" = "↓");
+    add_alias!("Up" = "↑");
+    add_alias!("NumLock" = "num_lock", "num");
+    add_alias!("KpSlash" = "kp_slash", "psls");
+    add_alias!("KpAsterisk" = "kp_asterisk", "past");
+    add_alias!("KpMinus" = "kp_minus", "pmns");
+    add_alias!("KpPlus" = "kp_plus", "ppls");
+    add_alias!("KpEnter" = "kp_enter", "pent");
+    add_alias!("KpDecimal" = "kp_decimal", "pdec");
+    add_alias!("Kp0" = "kp_0");
+    add_alias!("Kp0" = "kp0");
+    add_alias!("Kp1" = "kp_1");
+    add_alias!("Kp1" = "kp1");
+    add_alias!("Kp2" = "kp_2");
+    add_alias!("Kp2" = "kp2");
+    add_alias!("Kp3" = "kp_3");
+    add_alias!("Kp3" = "kp3");
+    add_alias!("Kp4" = "kp_4");
+    add_alias!("Kp4" = "kp4");
+    add_alias!("Kp5" = "kp_5");
+    add_alias!("Kp5" = "kp5");
+    add_alias!("Kp6" = "kp_6");
+    add_alias!("Kp6" = "kp6");
+    add_alias!("Kp7" = "kp_7");
+    add_alias!("Kp7" = "kp7");
+    add_alias!("Kp8" = "kp_8");
+    add_alias!("Kp8" = "kp8");
+    add_alias!("Kp9" = "kp_9");
+    add_alias!("Kp9" = "kp9");
+    add_alias!("Kp0" = "kp_0");
+    add_alias!("Kp0" = "kp0");
+    add_alias!("KpDot" = "kp_dot", "pdot");
+    add_alias!("KpEqual" = "kp_equal", "peql");
+    add_alias!("NonusBackslash" = "nonus_backslash", "nubs");
+    add_alias!("Application" = "app");
+    add_alias!("KbPower" = "kb_power");
+    add_alias!("F13");
+    add_alias!("F14");
+    add_alias!("F15");
+    add_alias!("F16");
+    add_alias!("F17");
+    add_alias!("F18");
+    add_alias!("F19");
+    add_alias!("F20");
+    add_alias!("F21");
+    add_alias!("F22");
+    add_alias!("F23");
+    add_alias!("F24");
+    add_alias!("Execute" = "exec");
+    add_alias!("Help");
+    add_alias!("Menu");
+    add_alias!("Select" = "slct");
+    add_alias!("Stop");
+    add_alias!("Again" = "agin");
+    add_alias!("Undo");
+    add_alias!("Cut");
+    add_alias!("Copy");
+    add_alias!("Paste" = "pste");
+    add_alias!("Find");
+    add_alias!("KbMute" = "kb_mute", "mute"); // FIXME: mismatch with QMK. `mute` in QMK is mapped to `KC_AUDIO_MUTE`. The same for the other `Kb*` keys.
```

Comment:
Should I change this?

![image](https://github.com/user-attachments/assets/ca11f101-7a0e-4f74-b245-c55d5985504f)

Reply:
Is it fixed?

Reply:
I have made changed on my local machine, waiting on finishing the others one to push it onto Github.